pallets/author-mapping/src/migrations.rs

```diff
@@ -112,112 +107,112 @@ impl<T: Config> OnRuntimeUpgrade for AddKeysToRegistrationInfo<T> {
-		for (author_id, account_id) in stored_data {
-			MappingWithDeposit::<T>::insert(author_id, account_id);
-		}
-
-		log::info!(target: "TwoXToBlake", "almost done");
-
-		// Return the weight used. For each migrated mapping there is a red to get it into
-		// memory, a write to clear the old stored value, and a write to re-store it.
-		let db_weights = T::DbWeight::get();
-		migrated_count.saturating_mul(2 * db_weights.write + db_weights.read)
-	}
-
-	#[cfg(feature = "try-runtime")]
-	fn pre_upgrade() -> Result<(), &'static str> {
-		use frame_support::{storage::migration::storage_iter, traits::OnRuntimeUpgradeHelpersExt};
-
-		let pallet_prefix: &[u8] = b"AuthorMapping";
-		let storage_item_prefix: &[u8] = b"MappingWithDeposit";
-
-		// We want to test that:
-		// There are no entries in the new storage beforehand
-		// The same number of mappings exist before and after
-		// As long as there are some mappings stored, one representative key maps to the
-		// same value after the migration.
-		// There are no entries in the old storage afterward
-
-		// Assert new storage is empty
-		// Because the pallet and item prefixes are the same, the old storage is still at this
-		// key. However, the values can't be decoded so the assertion passes.
-		assert!(MappingWithDeposit::<T>::iter().next().is_none());
-
-		// Check number of entries, and set it aside in temp storage
-		let mapping_count =
-			storage_iter::<RegistrationInfo<T>>(pallet_prefix, storage_item_prefix).count() as u64;
-		Self::set_temp_storage(mapping_count, "mapping_count");
-
-		// Read an example pair from old storage and set it aside in temp storage
-		if mapping_count > 0 {
-			let example_pair = storage_key_iter::<NimbusId, RegistrationInfo<T>, Twox64Concat>(
-				pallet_prefix,
-				storage_item_prefix,
-			)
-			.next()
-			.expect("We already confirmed that there was at least one item stored");
-
-			Self::set_temp_storage(example_pair, "example_pair");
-		}
-
-		Ok(())
-	}
-
-	#[cfg(feature = "try-runtime")]
-	fn post_upgrade() -> Result<(), &'static str> {
-		use frame_support::traits::OnRuntimeUpgradeHelpersExt;
-
-		// Check number of entries matches what was set aside in pre_upgrade
-		let old_mapping_count: u64 = Self::get_temp_storage("mapping_count")
-			.expect("We stored a mapping count; it should be there; qed");
-		let new_mapping_count = MappingWithDeposit::<T>::iter().count() as u64;
-		assert_eq!(old_mapping_count, new_mapping_count);
-
-		// Check that our example pair is still well-mapped after the migration
-		if new_mapping_count > 0 {
-			let (account, original_info): (NimbusId, RegistrationInfo<T>) =
-				Self::get_temp_storage("example_pair").expect("qed");
-			let migrated_info = MappingWithDeposit::<T>::get(account).expect("qed");
-			assert_eq!(original_info, migrated_info);
-		}
-
-		Ok(())
-	}
-}
+// /// Migrates the AuthorMapping's storage map fro mthe insecure Twox64 hasher to the secure
+// /// BlakeTwo hasher.
```

Comment:
I think we should just remove this if we don't ever expect it to come back. We can always retrieve it with git.

If we have a good reason to keep it, I say we use `/* */` instead of commenting out each line -- this will avoid polluting the commit history.

Reply:
I'll delete it